libs/tokens/src/const/tokensLists.ts

```diff
@@ -9,5 +9,6 @@ export const UNISWAP_TOKENS_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.
 export const GNOSIS_UNISWAP_TOKENS_LIST =
   'https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/GnosisUniswapTokensList.json'
 
-// TODO: add a uniswap mapped token list for arbitrum
-export const ARBITRUM_ONE_TOKENS_LIST = ''
+// TODO: update address once https://github.com/cowprotocol/token-lists/pull/393 is merged
+export const ARBITRUM_ONE_TOKENS_LIST =
+  'https://raw.githubusercontent.com/cowprotocol/token-lists/ca0fac81d0318234cf16b9f661d05a5c24c2ccb6/src/public/ArbitrumOneUniswapTokensList.json'
```

Comment:
Should into `develop` / `main` in the future?

Reply:
I would think `main`

Reply:
It will! As soon as https://github.com/cowprotocol/token-lists/pull/393 is merged